make/photon/registry/entrypoint.sh

```diff
@@ -7,10 +7,16 @@ if [ -d /etc/registry ]; then
 fi
 if [ -d /var/lib/registry ]; then
     chown 10000:10000 -R /var/lib/registry
-fi  
+fi
 if [ -d /storage ]; then
-    chown 10000:10000 -R /storage
-fi  
+    if ! ls -ld /storage | grep 'harbor harbor' ; then
```

Comment:
How about using `stat -c "%u"` ?

Reply:
sure, we can use 'stat -c "%u:%g" /data/registry'